integration/integration.cabal

```diff
@@ -25,6 +26,7 @@ common common-all
 
   default-extensions:
     NoImplicitPrelude
+    NoImportQualifiedPost
```

Comment:
This is required so that ormolo doesn't rewrite all of the imports and break haskell-src-exts' parsing.

This comment is moved around in sanitize-pr, and is otherwise very annoynig.